.ci/test_r_package_windows.ps1

```diff
@@ -36,12 +36,14 @@ if ($env:R_MAJOR_VERSION -eq "3") {
   # Rtools 3.x has to be installed at C:\Rtools\
   #     * https://stackoverflow.com/a/46619260/3986677
   $RTOOLS_INSTALL_PATH = "C:\Rtools"
-  $env:RTOOLS_MINGW_BIN = "$RTOOLS_INSTALL_PATH/mingw_64/bin"
+  $env:RTOOLS_MINGW_BIN = "$RTOOLS_INSTALL_PATH\mingw_64\bin"
+  $env:RTOOLS_PATHS = "$RTOOLS_INSTALL_PATH\usr\bin;" + "$env:RTOOLS_MINGW_BIN;"
   $env:RTOOLS_EXE_FILE = "rtools35-x86_64.exe"
   $env:R_WINDOWS_VERSION = "3.6.3"
 } elseif ($env:R_MAJOR_VERSION -eq "4") {
   $RTOOLS_INSTALL_PATH = "C:\rtools40"
-  $env:RTOOLS_MINGW_BIN = "$RTOOLS_INSTALL_PATH/mingw64/bin"
+  $env:RTOOLS_MINGW_BIN = "$RTOOLS_INSTALL_PATH\mingw64\bin"
+  $env:RTOOLS_PATHS = "$RTOOLS_INSTALL_PATH\bin;" + "$env:RTOOLS_MINGW_BIN;"
```

Comment:
ha! I did these backwards. Bad copy-paste, because this block is

```
if (rtools3) {
   ...
} elseif (rtool4) {
   ...
}
```

And the `r-lib/actions` on is

```
if (rtools4) {
   ...
} elseif (rtool4) {
   ...
}
```